pkg/server/server.go

```diff
@@ -197,15 +177,18 @@ func (s *Server) Write(ctx context.Context, req *openfgapb.WriteRequest) (*openf
 
 	storeID := req.GetStoreId()
 
-	modelID, err := s.resolveAuthorizationModelID(ctx, storeID, req.GetAuthorizationModelId())
+	// todo(jon-whit): read/write API endpoints (e.g. non-query endpoints) don't necessarily need
```

Comment:
We probably could avoid introducing the overhead of the model validation step in the read/write endpoints, but assuming the models are used across endpoints (which I assume they are), then it should be irrelevant anyways.. So I'm inclined to leave this as is and consistent with the rest, but I'm open to implementing another resolver that omits typesystem construction and validation and just resolves the model itself.

Thoughts? Concerns?

Reply:
I agree, leave as is.